.github/workflows/CI 7.0.yml

```diff
@@ -42,8 +70,16 @@ jobs:
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
           restore-keys: ${{ runner.os }}-composer-
 
+#      - name: Composer set symfony version (Symfony version ${{ matrix.symfony-version }})
+#        run: composer config extra.symfony.require "${{ matrix.symfony-version }}"
+
       - name: Composer update (Symfony version ${{ matrix.symfony-version }})
-        run: composer install --prefer-dist --no-interaction
+        run: |
+          composer global config --no-plugins allow-plugins.symfony/flex true
+          composer global require --no-progress --no-scripts --no-plugins symfony/flex
```

Comment:
I think you can add `--no-update` here so the update happens in the next line only.